test/check-machines-disks

```diff
@@ -416,6 +422,23 @@ class TestMachinesDisks(VirtualMachinesCase):
                 b.wait_visible("div.pf-c-modal-box button[aria-expanded=true]:contains(Hide additional options)")
                 b.select_from_dropdown(f"div.pf-c-modal-box #vm-{self.vm_name}-disks-adddisk-bus-type", self.bus_type)
 
+            # Configure serial number
+            if self.serial and b.is_visible("div.pf-c-modal-box button:contains(Show additional options)"):
```

Comment:
I don't like the `b.is_visible` here -- it should not be a surprise whether or not this is visible, but predictable. Ideally we could just drop this, what happens if you do?

Reply:
Right, looking at the code, it should be always present